src/writer/blocks/__init__.py

```diff
@@ -63,3 +64,4 @@
 WriterWebSearch.register("blueprints_writerwebsearch")
 WriterKeyValueStorage.register("blueprints_writerkeyvaluestorage")
 WriterChatReplyWithToolConfig.register("blueprints_writerchatreplywithtoolconfig")
+SharedBlueprint.register("shared_blueprint")
```

Comment:
We have a lot of frontend checks that evaluate `nodeType.startsWith('blueprints_')`. So, I'll suggest

```suggestion
BlueprintShared.register("blueprints_shared")
```